config/locales/en.yml

```diff
@@ -294,7 +294,11 @@ en:
   fatality_notice:
     alt_text: Ministry of Defence crest
     field_of_operation: Field of operation
+    none_added:
     operations_in: Operations in %{location}
+  field_of_operation:
+    context: British fatalities
+    title_start: Operations in
```

Comment:
Could we change this to being an interpolated value, and rename it to `title`? I think this was how it was done in Whitehall, and is a bit nicer than having half of the text here

Reply:
done :)